app-drawer/app-drawer.html

```diff
@@ -262,7 +262,7 @@
         // Only transition the drawer after its first render (e.g. app-drawer-layout
         // may need to set the initial opened state which should not be transitioned).
         this._styleTransitionDuration(0);
-        Polymer.RenderStatus.afterNextRender(this, function() {
+        Polymer.RenderStatus.afterNextRender(() => {
```

Comment:
for consistency with the rest of the package it might be preferable to use ES5 function declaration here instead of arrow function